community.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: page
+title: Community
+permalink: /community/
+redirect_from: /community/forums/
+---
+
+<a name="collaboration"></a>
+### Collaboration
+
+- <a href="https://forums.fusetools.com" target="_blank">Forums</a>
+- <a href="https://slackcommunity.fusetools.com" target="_blank">Slack</a>
```

Comment:
We have a link to the forum at the top of each page, and to the slack at the bottom of each page. Do we really need to add more links to these?

Reply:
It just looks like room to grow and they both at the top but yeah, we could remove these and keep forum in header and slack at bottom.